src/main/java/org/folio/innreach/mapper/InnReachTransactionSortingParametersMapper.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.innreach.mapper;
+
+import org.folio.innreach.domain.entity.InnReachTransactionSortingParameters;
+import org.folio.innreach.dto.InnReachTransactionSortingParametersDTO;
+import org.mapstruct.Builder;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring", injectionStrategy = InjectionStrategy.CONSTRUCTOR, uses = MappingMethods.class, builder = @Builder(disableBuilder = true))
+public interface InnReachTransactionSortingParametersMapper {
+  InnReachTransactionSortingParameters toEntity(InnReachTransactionSortingParametersDTO dto);
```

Comment:
do we really need this conversion and `InnReachTransactionSortingParameters` class at all? this is not an entity, right?
looks like the objects are completely identical: have the same fields and field types

Reply:
This mapper and `InnReachTransactionSortingParameters` entity were added to convert between domain's `TransactionType`/`TransactionState` and DTO's enums.